pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -50,8 +50,25 @@ type ProxySpec struct {
 
 // This is the struct that will contain information pertinent to Log storage (Elasticsearch)
 type LogStoreSpec struct {
-	Type              LogStoreType `json:"type"`
-	ElasticsearchSpec `json:"elasticsearch,omitempty"`
+	Type                LogStoreType `json:"type"`
+	ElasticsearchSpec   `json:"elasticsearch,omitempty"`
+	RetentionPolicySpec `json:"retentionPolicy,omitempty"`
```

Comment:
I think `RetentionPolicySpec` should be nested under the `ElasticsearchSpec` as any future technology may not operate the same...

thoughts @jcantrill ?

Reply:
This is correct per the enhancement doc.  The definition of the retention is not specific to Elasticsearch; it just says retain Xdays of logs of type source (e.g. apps).